### ISSUE TITLE
Build Preview URL on a PR

### ISSUE THREAD

**Issue by edmundmiller** [BioJulia/posts#23]
Follow up to #22

**Comment by edmundmiller** [comment]
#24

**Comment by kescobo** [comment]
Should we revert the original build workflow until this is resolved?

For the record, I'm fine with you merging stuff to `master` on your own to test this stuff, as long as you're able to revert if a build goes awry. I don't think anything here is so critical that a few minutes of downtime is a problem, just don't kick off a build and go to bed :-D

**Comment by edmundmiller** [comment]
Roger that!

I don't think it needs reverting, I think it's still deploying on master fine, so there's no regression?

**Comment by kescobo** [comment]
Well, for folks that have write access and make a PR from a branch in this repo (as opposed to a fork), the build preview *was* working. It's not a huge deal (and I'm having a baby in 5 days, so not like I'm gonna be doing much in the next few weeks / months), but if other folks want it - it's something to be considered.

**Comment by edmundmiller** [comment]
Ah gotcha, so this was working previously but not now? Going off this example: https://github.com/BioJulia/biojulia.github.io/pull/24

Congrats! Take care and enjoy your time!

**Comment by kescobo** [comment]
Yeah, that PR was after this change. See here https://github.com/BioJulia/biojulia.github.io/pull/17#issuecomment-1668049888 - it was a brief, glorious moment where it was working :laughing:
